source/expressions.tex

```diff
@@ -2990,8 +2992,6 @@
 \end{codeblock}
 (including in the case where $n$ is zero).
 \end{example}
-\end{itemize}
```

Comment:
You're now moving the example to the top level, as opposed to the outer bullet.

Reply:
Heh, one too many `\end{itemisze}`s that I moved? Changed to just move one.